### PR TITLE
Add test cases and fix some errors

### PR DESCRIPTION
**Pull request by Sata51**

### CODE REVIEW

File: src/bot/commands/crypto.js [hunk 1]
```diff
@@ -1,21 +1,49 @@
...
 
   try {
     const { base, price: cost, target } = await price.getCryptoPrice(currency, cryptoCurrency)
 
-    return `${num} ${base} is worth ${Intl.NumberFormat(currency === 'EUR' ? 'fr-FR' : 'en-US', {
+    if (currencies.indexOf(target) === -1) {
+      return `${quantity} ${base} is worth ${(cost * quantity).toFixed(2)} ${target}`
+    }
+
+    return `${quantity} ${base} is worth ${Intl.NumberFormat(currency === 'EUR' ? 'fr-FR' : 'en-US', {
```
Comment (mindsers) [inline src/bot/commands/crypto.js]: Could we extract the “formatting depending on the existence of a currency” in an external function?

File: src/bot/lib/utils/currencies.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
+export const currencies = [
```
Comment (mindsers) [inline src/bot/lib/utils/currencies.js]: Idk if this is possible but could we get this array from `Intl`? Like the supported list of currencies

Comment (Sata51) [reply src/bot/lib/utils/currencies.js]: It's in the spec but not currently implemented

Referenced here
https://github.com/tc39/proposal-intl-enumeration and https://stackoverflow.com/a/58940564/1897184

File: src/bot/commands/crypto.js (hunk 1 above)
Comment (Sata51) [reply src/bot/commands/crypto.js]: Formatting has been moved into `lib/utils/currencies.js` as required.

Jest is now part of the project like described in #7 and started in #27  . 

The computing and formatting test suite has been done.

We need to improve repository by adding jest testing in actions

**Review by mindsers — APPROVED** [review]
🚀

File: src/bot/commands/crypto.js (hunk 1 above)
Comment (mindsers) [reply src/bot/commands/crypto.js]: Awesome!

**Pull request merged**